Changes from dry run

MarionBWeinzierl opened the pull request:
Closes #14 . Implement some of the changes from the dry run (see issue)

MarionBWeinzierl committed:
removed some slides and implemented other changes from dry run

MarionBWeinzierl commented on slides/python.qmd:
```diff
@@ -245,21 +244,21 @@ Get in touch:
 {{< fa brands github >}} \ [MarionBWeinzierl](https://github.com/MarionBWeinzierl)
 
 {{< fa brands mastodon >}} \ [\@MarionBWeinzierl\@mast.hpc.social](https://mast.hpc.social/@MarionBWeinzierl)
+
```
This change is just to that the full contact info for everyone is visible and not hidden by the logo

jatkinson1000 commented on slides/_comments_docstrings.qmd:
```diff
@@ -180,84 +176,6 @@ def calculate_gyroradius(mass, v_perp, charge, B, gamma=None):
 ::::
 
 
```
I feel like commenting out with a `<!-- -->` could be preferable so the content is not 'lost' as you do below.

MarionBWeinzierl replied:
OK, happy to do that -- I thought I'd remove this one fully, as it it deprecated and therefore will probably not be taught in future

MarionBWeinzierl replied:
done

jatkinson1000 reviewed (commented):
Thanks @MarionBWeinzierl a comment in there.

Also, I forget the exact discussion, but on reflection here I would definitely cut the 'config file' part as this is just TMI at the end of a packed session and never lands very well.
Leaving the slide in to be visible is an option, but I am certain I would not cover it.
If anything I think fstrings would be easier for people to grasp if we somehow find ourselves with extra time.

MarionBWeinzierl commented:
> Thanks @MarionBWeinzierl a comment in there.
> 
> Also, I forget the exact discussion, but on reflection here I would definitely cut the 'config file' part as this is just TMI at the end of a packed session and never lands very well. Leaving the slide in to be visible is an option, but I am certain I would not cover it. If anything I think fstrings would be easier for people to grasp if we somehow find ourselves with extra time.

Any preferences about leaving the slides in and skipping them, or commenting them out for the case you have time?

MarionBWeinzierl committed:
comment pydocstyle part out rather than removing

MarionBWeinzierl committed:
also remove config file part

MarionBWeinzierl commented:
> > Thanks @MarionBWeinzierl a comment in there.
> > Also, I forget the exact discussion, but on reflection here I would definitely cut the 'config file' part as this is just TMI at the end of a packed session and never lands very well. Leaving the slide in to be visible is an option, but I am certain I would not cover it. If anything I think fstrings would be easier for people to grasp if we somehow find ourselves with extra time.
> 
> Any preferences about leaving the slides in and skipping them, or commenting them out for the case you have time?

For the time being I have removed both. However, not sure whether the title of the section is then still fitting (maybe with the "honorable mentions" it is), or whether it should be changed. Or another slide added to mention the stuff that we removed.

jatkinson1000 reviewed (approved):
Punch it Chewie.

@MarionBWeinzierl I can't see we will have lots of spare time, so both removed is probably fine. We can always mention or demo if we feel the need to.

At this point let's get something deliverable for the summer school. We can refine and add back at a later date after next week if so desired.
